packages/client/src/lib.js

```diff
@@ -284,7 +302,53 @@ class Web3Storage {
   }
 
   /**
-   * Fetch the Content Addressed Archive by it's root CID.
+   * Uploads CAR files to web3.storage. Files are hashed in the client and uploaded as a single
+   * [Content Addressed Archive(CAR)](https://github.com/ipld/specs/blob/master/block-layer/content-addressable-archives.md).
```

Comment:
Not hashed by the client here, the hashing and CAR construction needs to be done in the application.

```suggestion
   * Uploads a CAR ([Content Addressed Archive](https://github.com/ipld/specs/blob/master/block-layer/content-addressable-archives.md)) file to web3.storage.
```